hypothesis-python/src/hypothesis/core.py

```diff
@@ -852,7 +852,7 @@ def wrapped_test(*arguments, **kwargs):
             test = wrapped_test.hypothesis.inner_test
 
             if getattr(test, "is_hypothesis_test", False):
-                note_deprecation(
+                raise InvalidArgument(
                     (
                         "You have applied @given to test: %s more than once. In "
                         "future this will be an error. Applying @given twice "
```

Comment:
Now that this is an error, we can lose “In future this will be an error”.

Reply:
Thanks for spotting it!